corehq/apps/translations/app_translations/upload_form.py

```diff
@@ -255,7 +255,7 @@ def _check_for_shadow_form_warning(form):
 def escape_output_value(value):
     try:
         return etree.fromstring("<value>{}</value>".format(
-            re.sub("(?<!/)>", "&gt;", re.sub("<(\s*)(?!output)", "&lt;\\1", value))
+            re.sub(r"(?<!/)>", "&gt;", re.sub("<(\s*)(?!output)", "&lt;\\1", value))
```

Comment:
I was curious what you were going to do with @mkangia's comment, @orangejenny, because I thought dropping the `r` prefixes was fine. :grin:

@mkangia you probably know this already, but just to confirm we're on the same page, the `r` prefix just means "[all backslashes are left in the string](https://docs.python.org/2.0/ref/strings.html)". The first pattern has no backslashes, so the `r` has no effect, and the second pattern has one, but `\s` is not an escape sequence, so the `r` has no effect there either.

... I was going to leave this comment at that, but I went looking for what Python 3 docs said about raw string literals, and kept coming across regex patterns that had `r` prefixes even though they clearly didn't need them because they didn't have any backslashes. [Then I found](https://docs.python.org/3/library/re.html#regular-expression-syntax) "... it’s highly recommended that you use raw strings for all but the simplest expressions". Hmmm. Maybe this is a convention that GitHub's syntax highlighting assumes and I was just unaware of. (In my defence, PyCharm uses regex syntax highlighting for all patterns, regardless of whether they are raw string literals or not.)

Anyway, fwiw, I'm fine with these patterns as they are. I'm also fine without the `r` prefix because they don't need them. But if using raw strings for all regex patterns is a thing I didn't know about, we should probably just always use them.

Reply:
Thanks for sharing that @kaapstorm. I just choose to not modify what i don't understand completely or whose intent is not clear to me. After reading your note i confused myself with somethings i tried with re.sub, so i am going to leave this to Jenny to decide to change or not.

Reply:
Based on https://github.com/dimagi/commcare-hq/pull/23403 my understanding is that our convention is to use the `r` prefix for all regular expressions. @nickpell is that accurate?